docs/guides/retrieving_records.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: home
+title: Retrieving a single record
+parent: How-to guides
+nav_order: 3
+---
+
+## Retrieving a single record
+
+
+In addition to searching, it is also possible to retrieve a specific record. To do so, you will need the
+`timdexRecordId` for the item. You can find this by returning that field in a query, like so:
+
+```graphql
+{
+  search(searchterm: "pragmatic programmer") {
+    records {
+      title
+      timdexRecordId
+    }
+  }
+}
+
+```
+
+Once you have the `timdexRecordId` of the record in question, you can retrieve detailed information about that record:
+
+```graphql
+{
+  recordId(id: "alma:9935059777706761") {
+    title
+    edition
+    citation
+    holdings {
+      location
+    }
+    identifiers {
+      kind
+      value
+    }
+  }
+}
+```
+
```

Comment:
Maybe include something like:

"You can instead [request all of the fields](returning_fields) you want in the original search query instead of doing this multistep process if it makes more sense for your use case." ?

Reply:
Are all of those fields also available in `search`?

Reply:
Yup. Every field is available to be requested in Search.